web/js/containers/sidebar/layer-row.js

```diff
@@ -78,7 +78,25 @@ function LayerRow (props) {
   } = props;
 
   const encodedLayerId = util.encodeId(layer.id);
+  const { title } = names;
+  const removeLayerBtnId = `close-${compareState}${encodedLayerId}`;
+  const removeLayerBtnTitle = 'Remove layer';
```

Comment:
I'd suggest we use title case for these, e.g. "Remove Layer", "View Options", "View Description"

Reply:
Also, for ordering the menu items I'd suggest the same order they show left-to-right at desktop size which would be: "View Description", "View Options", "Remove Layer"